### PR TITLE
docker-files metadata

### PR DESCRIPTION
**Pull request by GuyAfik**
initiate metadata file of all images that are used in content-repo for this point of time

### CODE REVIEW

**Comment by JudahSchwartz** [comment]
lets merge this in the same pr as the update script change

File: dockerfiles-metadata.json [hunk 1]
```diff
@@ -0,0 +1,1299 @@
+{
+    "docker_images":{
```
Comment (kobymeir) [inline dockerfiles-metadata.json:2]: @GuyAfik can we add a scheme to this file, explaining the fields? consult with @kgal-pan about it. he did something for pack_metadata.json.

File: dockerfiles-metadata.json [hunk 2]
```diff
@@ -0,0 +1,1299 @@
...
+          }
+       },
+       "archer":{
+          "1.0.0.270":{
+             "python_version":"3.7.3"
+          }
+       },
+       "powershell-teams":{
+          "1.0.0.22275":{
+             "python_version":""
```
Comment (kobymeir) [inline dockerfiles-metadata.json:1295]: why it's empty here? if it's not applicable for this image or python doesn't exist, maybe we can skip empty keys creation?

File: dockerfiles-metadata.json [hunk 3]
```diff
@@ -0,0 +1,1299 @@
...
+             "python_version":"3.9.6"
+          },
+          "1.0.0.27257":{
+             "python_version":"3.9.7"
+          },
+          "1.0.0.55864":{
+             "python_version":"3.10.11"
+          },
+          "1.0.0.67573":{
+             "python_version":""
```
Comment (kobymeir) [inline dockerfiles-metadata.json:162]: it's also empty here? and in some other places, why?

**Review by kobymeir — CHANGES_REQUESTED** [review]

File: dockerfiles-metadata.json (hunk 1 above)
Comment (kgal-pan) [reply dockerfiles-metadata.json:2]: https://github.com/demisto/content/pull/21405

**Comment by GuyAfik** [comment]
will implement it in the same PR along with schema @kobymeir @JudahSchwartz , closing it

**Pull request closed without merging**